cluster/sbin/collect-debug-artifacts.sh

```diff
@@ -212,17 +232,25 @@ function collect_on_remote {
   int_stack_dumps="$3"
   get_all="$4"
   collector_host="$5"
-  collector_dir="$6"
-  verbose="$7"
-  start_epoch="$8"
-  end_epoch="$9"
-
-  tmp_dir="$(mktemp -d "$data_dir/data.XXXX")"
-  retval=$?
-  if [ ! -d ${tmp_dir} ]; then
-    echo "FAILED TO CREATE tmp dir on ${host} at ${data_dir} with errno ${retval}"
-    exit 1
-  fi
+  verbose="$6"
+  start_epoch="$7"
+  end_epoch="$8"
+
+  # Create a .tmpcda dir if not exists else empty it
+  tmp_dir="$data_dir/.tmpcda"
+  if [ -d ${tmp_dir} ]; then
+    rm -rf ${tmp_dir}/*
+  else
+    mkdir -p $tmp_dir
```

Comment:
Shouldn't this be done on the remote server itself? Else you are assuming the working directory is available over NFS but for persistent files we recommend having them locally.